HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealthConfiguration.swift

```diff
@@ -50,6 +50,17 @@ public final class GiniHealthConfiguration: NSObject {
      */
     public var numberOfTimesOnboardingShareScreenShouldAppear = 3
```

Comment:
@razvancapra 
remember we've discussed it and decided to remove this customization option 
![image](https://github.com/gini/gini-mobile-ios/assets/3911816/68d8cb72-f9b7-4de1-b0cf-d4fe6e9b8921)